shared/locales/date/date.en-US.properties

```diff
@@ -221,17 +221,28 @@ firstDayOfTheWeek = 0
 #
 # If 12h time format is selected, then the %H will display 12h clock and
 # p will be ignored.
-timePickerOrder=HMp
+timePickerOrder = HMp
 
 # This variable will be used to select date picker order
 # Y - year
 # M - month
 # D - day
-datePickerOrder=MDY
+datePickerOrder = MDY
 
 # Character used in timePicker to separate hours from minutes
 hourMinutesSeparator=:
 
+# Separator used for list formatting. \u0020 represents a blank space
+# Note that the trailing space is meaningful - "X, Y" vs. "X,Y"
+listSeparator_middle =,\u0020
```

Comment:
it should be in moz_intl.properties, but I wanted to avoid having to add moz_intl.properties and then ask developers to include both, or migrate eveyrthing from date.properties to moz_intl.properties and cause more work for localizers :( (also, migrating would require patching many apps).

So I just sticked to using date.properties for mozIntl.